src/SGB_Command_Packet.md

```diff
@@ -8,30 +8,36 @@ Game Boy keyboard matrix (which still works).
 ## Transferring Bits
 
 A command packet transfer must be initiated by setting both P14 and P15
-to LOW, this will reset and start the SNES packet receiving program.
+to LOW; this will reset and start the ICD2 packet receiving circuit.
 Data is then transferred (LSB first), setting P14=LOW will indicate a
 "0" bit, and setting P15=LOW will indicate a "1" bit. For example:
 
 ```
     RESET  0   0   1   1   0   1   0
 P14  --_---_---_-----------_-------_--...
 P15  --_-----------_---_-------_------...
+Time   0        50        100       150
```

Comment:
```suggestion
       ↑        ↑         ↑         ↑
Time   0        50        100       150
```

Reply:
The arrows are supposed to be 50 M-cycles apart, with each character representing 5 M-cycles.

Reply:
Fixed, I think?